centreon/www/front_src/src/Dashboards/components/DashboardPlaylists/PlaylistsListing/useLoadPlaylists.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState, useEffect } from 'react';
+
+import { useAtomValue } from 'jotai';
+
+import { useFetchQuery } from '@centreon/ui';
+
+import {
+  limitAtom,
+  pageAtom,
+  sortFieldAtom,
+  sortOrderAtom,
+  searchAtom
+} from './atom';
+import { PlaylistListingType } from './models';
+import { buildlistPlaylistsEndpoint, listPlaylistsDecoder } from './api';
+import { formatListingData } from './utils';
+
+interface UseLoadPlaylists {
+  data?: PlaylistListingType;
+  loading: boolean;
+}
+
+const useLoadPlaylists = (): UseLoadPlaylists => {
+  const [isMounting, setIsMounting] = useState(true);
+
+  const page = useAtomValue(pageAtom);
+  const limit = useAtomValue(limitAtom);
+  const sortField = useAtomValue(sortFieldAtom);
+  const sortOrder = useAtomValue(sortOrderAtom);
+  const searchValue = useAtomValue(searchAtom);
+
+  const sort = { [sortField]: sortOrder };
+  const search = {
+    regex: {
+      fields: ['name'],
+      value: searchValue
+    }
+  };
+
+  const { data, isLoading: loading } = useFetchQuery<PlaylistListingType>({
+    decoder: listPlaylistsDecoder,
+    getEndpoint: () => {
+      return buildlistPlaylistsEndpoint({
+        limit: limit || 10,
+        page: page || 1,
+        search,
+        sort
+      });
+    },
+    getQueryKey: () => ['playlists', sortField, sortOrder, page, limit, search],
+    queryOptions: {
+      suspense: isMounting
```

Comment:
Why ?

Reply:
If I set it to true, it will load the entire page with every request, including search, pagination, and sorting. The desired behavior is to trigger loading only during the mounting phase, allowing us to decide which component to display: the welcome page or the listing